### ISSUE TITLE
JSON API returns invalid JSON

### ISSUE THREAD

**Issue by karihaa** [jomjol/AI-on-the-edge-device#598]
Hi

In my system it seems the /json api sometimes returns not valid json. As follows:
```
{
"main":
  {
    "value": 35N.0758,
    "raw": "35N.0758",
    "error": "",
    "rate": "",
    "timestamp": "2022-01-23T10:57:48"
  }
}
```

Here the value tag is a string, so it should be quoted. Or actually, since there is a digit recognition error, I believe the value should be omitted - leaving just double quotes in there.
I have deactivated the PreValue, since it does not suit my need completely. 
Software version is `master - v10.1.0 - 2022-01-09`

Thank You for marvelous software!

**Comment by jomjol** [comment]
Your right, this is a not detected cause after a startup with no prevalue. I need to check in one of the next versions.

**Comment by jomjol** [comment]
This will be resolved in the next version v10.3
* in case there is still a "N", there will be an empty value.
